src/Components/pages/Config.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import {fire, auth, firestore} from '../../firebase.js';
+import {setUpRoom,getIndustries,getSectors,getMarketCaps,initSymbols} from '../firebase-access.jsx'
+import AsyncSelect from 'react-select/async';
+import Select from 'react-select'
+
+const ROUNDS = [
+    { value: 1, label: '1 Round' },
+    { value: 2, label: '2 Rounds' },
+    { value: 3, label: '3 Rounds' },
+    { value: 4, label: '4 Rounds' },
+    { value: 5, label: '5 Rounds' },
+    { value: 6, label: '6 Rounds' },
+    { value: 7, label: '7 Rounds' },
+  ]
+class Config extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+          Industry:null,
+          MarketCap:null,
+          Sector:null,
+          Rounds:null
+        };
+        this.submitHandler = this.submitHandler.bind(this);
+        this.getIndList = this.getIndList.bind(this);
+        this.getSectorList = this.getSectorList.bind(this);
+        this.getMarketCapList = this.getMarketCapList.bind(this);
+      }
+      // async componentDidMount() {  
+      //     this.setState({
+      //         Industries: await getIndustries(firestore),
+      //         Sectors: await getSectors(firestore),
+      //         MarketCaps: await getMarketCaps(firestore),
+      //     });
+      // }
+  
+      getIndList = async()=>{
+         
+          let IndList =await getIndustries(firestore);
+          return IndList;
+      }
+      getSectorList = async()=>{
+         
+          let SectorList =await getSectors(firestore);
+          return SectorList;
+      }
+      getMarketCapList = async()=>{
+         
+          let CapList =await getMarketCaps(firestore);
+          return CapList;
+      }
+      handleIndustryChange = selectedOption => {
+          this.setState({ Industry: selectedOption.value });
+          console.log(`Industry selected:`, selectedOption);
+      };
+  
+      handleSectorChange = selectedOption => {
+          this.setState({ Sector: selectedOption.value });
+          console.log(`Sector selected:`, selectedOption);
+      };
+  
+      handleMarketCapChange = selectedOption => {
+          this.setState({ MarketCap: selectedOption.value });
+          console.log(`MarketCap selected:`, selectedOption);
+      };
+      handleRoundChange = selectedOption =>{
+        this.setState({ Rounds: selectedOption.value });
+        console.log(`Rounds selected:`, selectedOption);
+      }
+      
+      submitHandler = async (event) => {
+          event.preventDefault(); // prevent page 
+          const data = new FormData(event.target);
+          console.log(data);
+          const Industry = this.state.Industry;
+          const Sector = this.state.Sector;
+          const MarketCaps = this.state.MarketCap;
+          const Rounds = this.state.Rounds;
+          let num = parseInt(data.get("NumSymbols"), 10);

```

Comment:
```suggestion

          let num = parseInt(data.get("NumSymbols"), 10);
```

add in whitespace, otherwise these chunks get really huge